src/style.css

```
@@ -1 +1,70 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
```

```suggestion
@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");

```

**and connect fonts via link in the `<head>`**